### PR TITLE
Add packages for ISO screen reading

### PR DESCRIPTION
**Pull request by thcrain-msft**
Reviewer Notes: 
- This is going into the `thcrain/a11y` branch, which is based on latest 1.0-dev with commits from #601

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Adds screen reader packages to enable accessibility in the ISO installer.

###### Change Log  

- Add espeakup, espeak-ng, pcaudiolib packages

###### Does this affect the toolchain?  

NO

###### Test Methodology

- Local build

### CODE REVIEW

**Review by jslobodzian — APPROVED** [review]
:shipit:

File: SPECS/espeak-ng/espeak-ng.spec [hunk 1]
```diff
@@ -0,0 +1,77 @@
+Summary:        Compact text-to-speech synthesizer
+Name:           espeak-ng
+Version:        1.50
+Release:        1%{?dist}
+# Apache2 license applies only to Android APK code- does not apply here
+# BSD license applies only to Windows code- does not apply here
+License:        GPLv3 and Unicode
```
Comment (PawelWMS) [inline SPECS/espeak-ng/espeak-ng.spec:7]: Nit: I think "and" and "or" are supposed to be all-caps in the `License` tag.

File: SPECS/espeak-ng/espeak-ng.spec [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+%defattr(-,root,root)
+%{_includedir}/espeak
+%{_includedir}/espeak-ng
+%{_libdir}/pkgconfig/*.pc
+%{_libdir}/*.a
+%{_libdir}/*.so
+
+%changelog
+* Fri Feb 05 2021 Thomas Crain <thcrain@microsoft.com> - 1.50-1
+- Original version for CBL-Mariner (license: MIT)
```
Comment (PawelWMS) [inline SPECS/espeak-ng/espeak-ng.spec:77]: Please verify the package's license and indicate it in the changelog by saying "License verified" or similar.

File: SPECS/espeakup/espeakup.spec [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+%license COPYING
+%doc ChangeLog README
+%{_bindir}/%{name}
+%{_libdir}/systemd/system/%{name}.service
+%{_libdir}/modules-load.d/%{name}.conf
+%{_mandir}/man8/*
+
+%changelog
+* Thu Jan 07 2021 Thomas Crain <thcrain@microsoft.com> - 0.80-1
+- Original version for CBL-Mariner (license: MIT)
```
Comment (PawelWMS) [inline SPECS/espeakup/espeakup.spec:56]: Same comment about license verification.

File: SPECS/pcaudiolib/pcaudiolib.spec [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+%files devel
+%defattr(-,root,root)
+%{_includedir}/%{name}/audio.h
+%{_libdir}/libpcaudio.a
+%{_libdir}/libpcaudio.so
+
+%changelog
+* Fri Feb 05 2021 Thomas Crain <thcrain@microsoft.com> - 1.1-1
+- Original version for CBL-Mariner (license: MIT)
```
Comment (PawelWMS) [inline SPECS/pcaudiolib/pcaudiolib.spec:70]: Same comment about license verification.

File: toolkit/resources/imageconfigs/packagelists/accessibility-packages.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+    "packages": [
+        "espeak-ng",
+        "espeakup",
+        "kernel-drivers-sound",
```
Comment (PawelWMS) [inline toolkit/resources/imageconfigs/packagelists/accessibility-packages.json:5]: Nit: can we switch places of the drivers to maintain alphabetical order?:)

**Review by PawelWMS — CHANGES_REQUESTED** [review]

File: toolkit/resources/imageconfigs/packagelists/accessibility-packages.json (hunk 5 above)
Comment (thcrain-msft) [reply toolkit/resources/imageconfigs/packagelists/accessibility-packages.json]: Removed from this PR, but will address, thanks!

File: SPECS/espeak-ng/espeak-ng.spec (hunk 1 above)
Comment (thcrain-msft) [reply SPECS/espeak-ng/espeak-ng.spec]: Resolved, thanks

File: SPECS/espeak-ng/espeak-ng.spec (hunk 2 above)
Comment (thcrain-msft) [reply SPECS/espeak-ng/espeak-ng.spec:77]: Resolved, thanks

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**